### ISSUE TITLE
prometheus-to-sd deprecation?

### ISSUE THREAD

**Issue by logicbomb421** [GoogleCloudPlatform/k8s-stackdriver#343]
Hello,

I just landed on this project again after a few months away and notice the big message at the top of the README basically saying _not_ to use this utility anymore, and to instead prefer Google Cloud Monitoring.

I am slightly confused, as I have successfully used this in the past to export some RabbitMQ metrics to Stackdriver, which was _incredibly_ handy because it meant I did not need to run my own Prometheus system.

Now looking at the alternative to this, I see we _do_ need to be running our own Prometheus instance now? It seems this incredibly simple, easy to configure tool has been replaced in favor of a much more robust, but also much more complex alternative?

I am in a position where I want to eventually run my own Prometheus system, but do not have the time now and was planning on using prometheus-to-sd in the interim, since I have a lot of monitoring already configured in Stackdriver.

Just trying to understand the goal here.

**Comment by huydinhle** [comment]
Yeah, it seems like no one is responding to this repo any more

**Comment by ocervell** [comment]
@logicbomb421 @huydinhle  yes this tool is for internal use only. You can use the OpenTelemetry collector deployed as a daemonset to scrape Prometheus metrics, see an example here: https://github.com/ocervell/gunicorn-opentelemetry-poc/blob/master/k8s/otel-agent.yaml
